stories/__tests__/SQFormDatePicker.stories.test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {LocalizationProvider} from '@material-ui/pickers';
+import MomentAdapter from '@material-ui/pickers/adapter/moment';
+import {composeStories} from '@storybook/testing-react';
+import {render, screen, within} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as stories from '../SQFormDatePicker.stories';
+
+const {BasicDatePicker} = composeStories(stories);
+
+const renderDatePicker = props => {
+  render(
+    <LocalizationProvider dateAdapter={MomentAdapter} locale={'en'}>
+      <BasicDatePicker {...props} />
+    </LocalizationProvider>
+  );
+};
+
+describe('SQFormDatePicker Tests', () => {
+  it('should render a label and input', () => {
+    renderDatePicker();
+
+    const label = screen.getByText('Date');
+    expect(label).toBeInTheDocument();
+    expect(label).toHaveTextContent('Date');
+
+    const textField = screen.getByRole('textbox', {name: /date/i});
+    expect(textField).toBeInTheDocument();
+    expect(textField).toHaveValue('09/22/2020');
+  });
+
+  it('should render default with empty initial value', () => {
+    const initialValues = {
+      date: ''
+    };
+
+    renderDatePicker({SQFormProps: {initialValues}});
+
+    const textField = screen.getByRole('textbox', {name: /date/i});
+    expect(textField).toHaveValue('');
+  });
+
+  it('should render value with non-empty initial value', () => {
+    renderDatePicker();
+
+    const textField = screen.getByRole('textbox', {name: /date/i});
+    expect(textField).toBeInTheDocument();
+    expect(textField).toHaveValue('09/22/2020');
+  });
+
+  it('should update when text is entered', () => {
```

Comment:
The content of this test seems like a duplicate of the second test but the name seems different from what's being tested here - are you just using this as a placeholder to fill the test since we can't fully test this until MUI v5?

Reply:
Ah, good catch. I should probably take this one out for now. I had a bunch of commented out items in here previously, so it doesn't make sense now.